src/presentation/http/router/note.ts

```diff
@@ -70,6 +69,24 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
           $ref: 'NoteSchema#/properties/id',
         },
       },
+      response: {
+        '2xx': {
+          type: 'object',
+          properties: {
+            note: {
+              $ref: 'NoteSchema#/properties/id',
```

Comment:
why just an `id`? There should be all public fields, so it's better to reference the whole `NoteSchema`